ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/emailinput/v1/emailinput/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,312 @@
+                                                <justify
+                                                        jcr:primaryType="nt:unstructured"
+                                                        items="[justify#justifyleft,justify#justifycenter,justify#justifyright]"
+                                                        ref="justify"/>
+                                                <lists
+                                                        jcr:primaryType="nt:unstructured"
+                                                        items="[lists#unordered,lists#ordered,lists#outdent,lists#indent]"
+                                                        ref="lists"/>
+                                            </popovers>
+                                        </inline>
+                                        <fullscreen
+                                                jcr:primaryType="nt:unstructured"
+                                                toolbar="[format#bold,format#italic,format#underline,subsuperscript#subscript,subsuperscript#superscript,-,links#modifylink,links#unlink,links#anchor,tracklinks#modifylinktracking,-,justify#justifyleft,justify#justifycenter,justify#justifyright,-,image#imageProps,-,lists#unordered,lists#ordered,lists#outdent,lists#indent,-,misctools#sourceedit,fullscreen#finish]">
+                                            <popovers jcr:primaryType="nt:unstructured">
+                                                <styles
+                                                        jcr:primaryType="nt:unstructured"
+                                                        items="styles:getStyles:styles-pulldown"
+                                                        ref="styles"/>
+                                                <paraformat
+                                                        jcr:primaryType="nt:unstructured"
+                                                        items="paraformat:getFormats:paraformat-pulldown"
+                                                        ref="paraformat"/>
+                                            </popovers>
+                                        </fullscreen>
+                                    </cui>
+                                </uiSettings>
+                            </richTextValue>
+
+                            <multiLine
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:hideResource="{Boolean}true"/>
+                            <allowRichText
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:hideResource="{Boolean}true">
+                            </allowRichText>
+                            <autocomplete
+                              jcr:primaryType="nt:unstructured"
+                              sling:hideResource="{Boolean}true">
+                                <options
```

Comment:
options not needed. But while autocomplete is removed from UI - do ensure correct autocomplete value is being set.